logpyle/__init__.py

```diff
@@ -389,7 +389,8 @@ def __init__(self, filename=None, mode="r", mpi_comm=None, capture_warnings=True
           If this database exists, the current state is loaded from it.
         :param mode: One of "w", "r" for write, read. "w" assumes that the
           database is initially empty. May also be "wu" to indicate that
-          a unique filename should be chosen automatically.
+          a unique filename should be chosen automatically. May also be "wo"
+          to indicate that the file should be overwritten.
```

Comment:
How is `"w"` different from `"wo"`?

Reply:
`"w"` will not overwrite an existing file if it already exists (see line 490), `"wo"` will.

Reply:
OIC